### PR TITLE
chore: Add Valinor ParamConverter (ported from 2.x)

### PR DESCRIPTION
**Pull request by fred-jan**
Change was ported from the `2.x` so that it can be used in our PHP 7.4 applications as well.

Should be tagged as `1.2`

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -4,7 +4,14 @@
     "type": "library",
     "license": "MIT",
     "config": {
-        "sort-packages": true
+        "platform": {
+            "php": "7.4.7"
+        },
```
Comment (frankverhoeven) [inline composer.json]: This shouldn't be here

**Review by frankverhoeven — APPROVED** [review]
One nit, otherwise LGTM

File: composer.json (hunk 1 above)
Comment (fred-jan) [reply composer.json]: Ah yes, remainder of my multiple-php-versions struggles, I'll remove it

File: tests/Symfony/Request/ParamConverter/StubValinorParamConverter.php [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    {
+        return StubId::class;
+    }
+
+    /**
+     * @inheritDoc
+     */
+    protected function getData(Request $request, ParamConverter $configuration)
+    {
+        /** @psalm-suppress InternalMethod */
```
Comment (frankverhoeven) [inline tests/Symfony/Request/ParamConverter/StubValinorParamConverter.php]:
```suggestion
        /** @psalm-suppress InternalMethod, ReservedWord */
```

**Pull request merged**